docs/source/cn/pooling.py

```diff
@@ -44,3 +44,244 @@
         - **output_size** (Int64List[3]): 目标输出大小（单个整数或包含三个整数的元组）
     """,
 )
+
+reset_docstr(
+    oneflow.nn.AdaptiveAvgPool1d,
+    r"""AdaptiveAvgPool1d(output_size)
+
+    在由多个平面组成的输入信号 `input` 上应用 1D 自适应平均池化。
+
+    对于任何大小的输入，输出大小都是 H 。
+    输出的数量等于输入平面的数量。
+
+    参数：
+        - **output_size** (Int64List[1]): 目标输出大小（单个整数）
```

Comment:
原英文是` output_size: the target output size H` 中文要不要加上这个 H
```suggestion
        - **output_size** (Int64List[1]): 目标输出大小 H（单个整数）
```